index.js

```diff
@@ -1,7 +1,57 @@
+// Students Ana and Robson
+
+console.log("I'm ready!");
+
 // Iteration 1: Names and Input
+let hacker1 = "Robson";
+let hacker2 = "Ana";
 
+console.log(`The driver's name is ${hacker1}`);
+console.log(`The navigator's name is ${hacker2}`);
 
 // Iteration 2: Conditionals
-
+if (hacker1.length > hacker2.length){
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters.`);
+}
+else if (hacker1.length < hacker2.length){
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`);
+}
+else if (hacker1.length = hacker2.length){
```

Comment:
When you want to compare something you need to use either `==` or `===` if you use just one equal sign `=` you are assigning a new value